.env

```diff
@@ -1,5 +1,5 @@
-TELEGRAM_BOT_TOKEN= # insert @BotFather token here
+TELEGRAM_BOT_TOKEN= 5523781029:AAEvkb0Cb904Yijt_v698ddhf77OnnBt78I
```

Comment:
never commit changes to this file. 

Now you exposed your API token and everyone on the internet can steal it